cython/gtsam/utils/visual_data_generator.py

```diff
@@ -99,11 +99,11 @@ def generate_data(options):
     r = 40
     for i in range(options.nrCameras):
         theta = i * 2 * np.pi / options.nrCameras
-        t = Point3(r * np.cos(theta), r * np.sin(theta), height)
-        truth.cameras[i] = PinholeCameraCal3_S2.Lookat(t,
-                                                       Point3(),
-                                                       Point3(0, 0, 1),
-                                                       truth.K)
```

Comment:
I guess these Python (and C++ below) fixes **should** ideally be in a separate PR, but probably you found and fix these errors while adding GitHub Actions, so... perhaps a good idea, once this PR builds, is to split all commits in this PR into two PRs: one fixing python/cpp code, then another (with all these small commits squashed) to add GH actions.

At least, that would be the best to keep git history as clean as possible ;-)

Reply:
I can do that. That's a great suggestion.

Reply:
The fix for the python/cpp code is part of another merge that happened a long time ago, so that may be a bit tricky to do. :(
If we squash and merge this PR, that should also fix the second issue, so I am going to leave this PR as is.